### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by renanborgez**
Installation, looks like something simple, but isn't.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 Integrate react-intl with gettext tools. This package provides a CLI tool to help converting between react-intl JSON files and gettext PO files.
 
+## Installation
+
+```npm install react-intl-gettext --save-dev```
```
Comment (rubencosta) [inline README.md]: - [ ] change markdown syntax to enable shell syntax highlighting
- [ ] add instructions for [yarn](https://yarnpkg.com/en/)

**Review by rubencosta — CHANGES_REQUESTED** [review]
@renanborgez you're right. There are only a couple of points that I would like to be addressed.

**Comment by renanborgez** [comment]
@rubencosta 
updated! 

:)

**Review by rubencosta — APPROVED** [review]

**Comment by rubencosta** [comment]
@renanborgez thanks for your contribution :)

**Pull request merged**